packages/graphql-typescript-definitions/README.md

```diff
@@ -78,7 +78,7 @@ This package will generate matching `.d.ts` files for each `.graphql` file you s
 
 ### Configuration
 
-This tool consumes a [`.graphqlconfig`](https://github.com/prisma/graphql-config) file, typically placed in the root of the project. This configuration file is compatible with the [VSCode extension](https://github.com/prisma/vscode-graphql) to provide syntax highlighting and autocomplete suggestions. The configuration file supports a single nameless project or multiple named projects, with each project linked to a schema and a set of include and exclude globbing patterns. Upon processing a schema file, the schema’s types will be extracted to `types.ts` (or `${projectName}-types.ts`, if the project is named) and written to the `schema-types-path` (or use the `schemaTypesPath` extension as an override). Note that your config should supply include and exclude globbing patterns as _relative_ paths (relative to the location of the `.graphqlconfig`).
+This tool consumes a [`.graphqlconfig`](https://github.com/prisma/graphql-config) file placed in the root of the project. This file is compatible with the [vscode-graphql extension](https://github.com/prisma/vscode-graphql). This provides syntax highlighting and autocomplete suggestions. The file may contain a single nameless project or more than one named projects. Each project links to a schema and a set of include and optional exclude globbing patterns. `include` and `exclude` globbing patterns match paths relative to the location of the `.graphqlconfig` file. This tool reads the `.graphqlconfig` file and load all linked schemas. The tool then extracts all enums, input objects, and custom scalars as schema types. The tool then writes the schema types to `types.ts` (or `${projectName}-types.ts` for named projects). The tool writes these files to the `schema-types-path` (or uses the `schemaTypesPath` extension as an override).
```

Comment:
The VSCode details break the flow of "there's a config file, here's what it looks like" sentences.
  It can be moved into a separate paragraph.

Reply:
`more than one` => `many`

Reply:
"links" isn't obvious in this context.  It'd be nice if the docs focused on what the tool does with schemas.

Reply:
I'd assume more knowledge of globs to make the middle part of this a bit more concise.

Reply:
For a person with my attention span, the last section is really hard to focus on.  I'd prefer some bullet points for my eyes to flit around 😬

Reply:
Ultimately, this would look something like...

-----

This tool reads schema information from a `.graphqlconfig` file in the project root.  The configuration can contain one, nameless project, or many named projects.

Each project specifies a `schemaPath`, and a set of include/exclude globs. Globs match paths relative to the location of the configuration file. 

On startup, this tool:
* Fetches all schemas
* Extracts all enums, input objects, and custom scalars as schema types
* Writes the schema types to `types.ts` (or `${projectName}-types.ts` for named projects)
  * Files are written to the `schema-types-path` 
  * Use a `schemaTypesPath` extension to override this behaviour

#### vscode integration
`.graphqlconfig` files are compatible with the `vscode-graphql extension`.  This extension provides syntax highlighting and autocomplete suggestions.

#### Examples
See the official specification documentation for more detail and examples.

Reply:
sounds good. this was a bit more formatting than i had originally anticipated, but might as well do it properly if I'm changing it in the first place.